src/styles/blocks/_menu-header.scss

```diff
@@ -0,0 +1,44 @@
+.menu-header {
+  &__list {
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+  }
+
+  &__item {
+    &:not(:last-child) {
+      margin-right: 20px;
+    }
+    text-transform: uppercase;
+  }
+
+  &__link {
+    display: block;
+    padding-bottom: 22.5px;
+    padding-top: 22.5px;
+    line-height: 125%;
+    position: relative;
+    transition: all $transition;
```

Comment:
[STYLES] - If you need to adjust positioning with 1px or use fractional numbers like 0.5 ANYWHERE, you are doing something wrong, change the alignment technique you use
Also, don't forget to specify to what properties the transition should apply as you do it before.

Reply:
what can i do if in design there is padding 22.5px